src/main/java/com/team4/testingsystem/entities/Question.java

```diff
@@ -0,0 +1,83 @@
+package com.team4.testingsystem.entities;
+
+import javax.persistence.*;
+
+@Entity
+public class Question {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(name = "question_body")
+    private String questionBody;
+    private int points;
+    @Column(name = "is_avaliable")
+    private int isAvailable;
+
+    @OneToOne
+    @JoinColumn(name = "id", insertable = false, updatable = false)
+    private User creatorUserId;
```

Comment:
The field is storing user, that created a question. Let`s call it something like "creator".
User - is in context.
Id - is about FK in a database. It is a separate layer and we don't need to propagate it to a system.